app/controllers/discourse_assign/assign_controller.rb

```diff
@@ -126,17 +126,27 @@ def group_members
         SQL
       end
 
+      group_assignment_count = Topic
+        .joins("JOIN assignments a ON a.topic_id = topics.id AND a.assigned_to_id IS NOT NULL ")
```

Comment:
`a.assigned_to_id IS NOT NULL` appears redundant because of the `where a.assigned_to_id = :group_id` below